src/codeu/chat/client/commandline/Chat.java

```diff
@@ -430,12 +494,21 @@ public void invoke(List<String> args) {
           }
         
           if (message.length() > 0) {
-            conversation.add(message);
+          	MessageContext m = conversation.add(message);
```

Comment:
It doesn't look like this `m` variable is actually used, so you should revert this line to how it was before.